composer.json

```diff
@@ -16,6 +16,7 @@
         "ext-json": "*",
         "acquia/drupal-environment-detector": "^1.2.0",
         "bugsnag/bugsnag": "^3.0",
+        "composer/composer": "2.5.8",
```

Comment:
Necessary to work around an issue where Composer is choosing a dev version of Composer in order to bump the major version of some other dependency